### ISSUE TITLE
DatePicker year/date button has insufficient contrast when not selected

### ISSUE THREAD

**Issue by jonahwilliams** [flutter/flutter#19944]

**Comment by jonahwilliams** [comment]
Duplicate of https://github.com/flutter/flutter/issues/14400
